test/api/test_rest.py

```diff
@@ -632,3 +635,16 @@ def test_known_issues_filter_by_environment(self):
         data = self.hit('/api/knownissues/?environment=%d' % env_id)
         self.assertEqual(1, len(data['results']))
         self.assertEqual('knownissue_bar', data['results'][0]['title'])
+
+    def test_project_compare_builds_with_finished_status_and_regressions(self):
+        foo_suite, _ = self.project.suites.get_or_create(slug='foo')
+        self.testrun4.tests.get_or_create(suite=foo_suite, name='dummy', result=True)
+        self.testrun6.tests.get_or_create(suite=foo_suite, name='dummy', result=False)
+        UpdateProjectStatus()(self.testrun4)
+        UpdateProjectStatus()(self.testrun6)
+        data = self.hit('/api/projects/%d/compare_builds/?baseline=%d&to_compare=%d' % (self.project.id, self.build4.id, self.build6.id))
+        self.assertEqual(1, len(data['regressions']['myenv']['foo']))
+
+    def test_project_compare_builds_with_non_finished_status(self):
```

Comment:
Please also add a test that tries to use versions instead of IDs for selecting a build.

Reply:
@mwasilew sure..make sense

Reply:
@mwasilew did you mean that the api should support both id and version for picking builds for comparison?

Reply:
no, just ID. The test should exercise the code that checks proper input.